### PR TITLE
Add connector (api_cv_attachment_srv)

### PR DESCRIPTION
**Pull request by osanda2024**
## Purpose
This pull request focuses on adding the connector (api_cv_attachment_srv) under the delivery and transportation module of SAP S/4 HANA.

## Checklist
- [ ] Add first connector
- [ ] Sanitize the Open API Specification
- [ ] Add operation ID to the Open API Specification
- [ ] Generate client using the Open API Specification
- [ ] Sanitize the generated client
- [ ] Generate mock server
- [ ] Add tests

### CODE REVIEW

File: ballerina/api_cv_attachment_srv/build.gradle [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ * under the License.
+ */
+import org.apache.tools.ant.taskdefs.condition.Os
+
+plugins {
+    id 'net.researchgate.release'
+    id 'io.ballerina.plugin'
+}
+
+description = 'Ballerina - SAP S/4HANA Freight Agreement Attachment API'
```
Comment (niveathika) [inline ballerina/api_cv_attachment_srv/build.gradle:25]: Docs and descriptions need to be reviewed again.

File: ballerina/api_cv_attachment_srv/docs.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "description": "The `ballerinax/sap.s4hana.api_del_doc_with_credit_block` package offers APIs for seamless integration with the [Delivery Document with Credit Block API v1.0.0](https://api.sap.com/api/API_DEL_DOC_WITH_CREDIT_BLOCK/overview). The service contains entities for credit blocked delivery document and reject reason. Once the delivery document has been checked, released, or rejected, a success message containing the document number is sent in the response.",
```
Comment (niveathika) [inline ballerina/api_cv_attachment_srv/docs.json]: Same

File: ballerina/api_cv_attachment_srv/modules/mock/api_cv_attachment_srv_mock_service.bal [hunk 3]
```diff
@@ -0,0 +1,64 @@
+// AUTO-GENERATED FILE.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+import ballerina/http;
+// import ballerina/os;
+
+// boolean isBalBuild = os:getEnv("IS_BAL_BUILD") == "true";
```
Comment (niveathika) [inline ballerina/api_cv_attachment_srv/modules/mock/api_cv_attachment_srv_mock_service.bal]: This needs to be reverted

File: ballerina/build.gradle [hunk 4]
```diff
@@ -24,6 +24,7 @@ description = 'Ballerina - SAP S/4HANA Delivery and Transporation APIs'
 subprojects {
     ext {
         packageOrg = "ballerinax"
+        ballerinaTomlFilePlaceHolder = new File("${project.rootDir}/build-config/resources/Ballerina.toml")
```
Comment (niveathika) [inline ballerina/build.gradle]: No need for this line

**Review by niveathika — CHANGES_REQUESTED** [review]

File: ballerina/build.gradle (hunk 4 above)
Comment (osanda2024) [reply ballerina/build.gradle]: Updated in [Update review changes](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/11/commits/efc0871542242b1e5b587b00abe2fd99465c9fbe)

File: ballerina/api_cv_attachment_srv/modules/mock/api_cv_attachment_srv_mock_service.bal (hunk 3 above)
Comment (osanda2024) [reply ballerina/api_cv_attachment_srv/modules/mock/api_cv_attachment_srv_mock_service.bal]: Updated in [Update review changes](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/11/commits/efc0871542242b1e5b587b00abe2fd99465c9fbe)

File: ballerina/api_cv_attachment_srv/build.gradle (hunk 1 above)
Comment (osanda2024) [reply ballerina/api_cv_attachment_srv/build.gradle:25]: Updated in [Update documentation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/11/commits/116c6665709fadd491c798aa5bb14999eefc57a1)

File: ballerina/api_cv_attachment_srv/docs.json (hunk 2 above)
Comment (osanda2024) [reply ballerina/api_cv_attachment_srv/docs.json]: Updated in [Update documentation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/11/commits/116c6665709fadd491c798aa5bb14999eefc57a1)

**Review by niveathika — DISMISSED** [review]

**Review by niveathika — APPROVED** [review]

**Pull request merged**